bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/DateVersionCommand.java

```diff
@@ -0,0 +1,121 @@
+package com.almightyalpaca.discord.jdabutler.commands.commands;
+
+import com.almightyalpaca.discord.jdabutler.Bot;
+import com.almightyalpaca.discord.jdabutler.commands.Command;
+import com.almightyalpaca.discord.jdabutler.util.EmbedUtil;
+import com.kantenkugel.discordbot.jenkinsutil.JenkinsApi;
+import com.kantenkugel.discordbot.jenkinsutil.JenkinsBuild;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+
+import java.awt.Color;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateVersionCommand extends Command
+{
+    private static final String DATE_FORMAT = "yyyy-MM-dd; HH:mm:ss";
```

Comment:
That reminds me... I think i added the DurationUtils i wrote to Butler... Which means we could print the difference version-release -> now in some 1w, 3d, ... format as well (next to datetime).
Possibly needs to be expanded with month/year handling cuz i don't think i did those

Reply:
@Almighty-Alpaca Which Timezone should I use then? Or do you mean I should just add `O` to the [Pattern](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)?
@kantenkugel Can you explain the `includeIntermed` param of the `formatDuration` function? I saw no difference when trying out both.

Currently it looks like this:
![current state](http://puu.sh/DNrEl/4521795fde.png)
But this doesn't look very good to me. Any suggestions or are you fine with that?

>That said I also like @kantenkugel's idea to add it to the `changelog` command

Should I add it to the Changelog Command as well or should this be handled in another PR?

Reply:
The `includeIntermed` parameter determines what happens to 0-values... with `includeIntermed` set to `true`, `10h, 0m 0s` is possible. With `includeIntermed` set to `false`, it would just print `10h`. `0s` is only printed if no other value is >0

Reply:
Agree with Kant about UTC.

Looking at the last screenshot though I think it would be kinda cool to make it more text like, for example:

**Version 3.8.3_464 was released on 22/06/2019 at 18:08 (GMT)**
That was approximately 4 Days, 3 Hours and 32 Minutes ago.

Reply:
Okay, UTC is already used by now, `includeIntermed` is also set to `true`.

> Looking at the last screenshot though I think it would be kinda cool to make it more text like, for example:
> 
> **Version 3.8.3_464 was released on 22/06/2019 at 18:08 (GMT)**
> That was approximately 4 Days, 3 Hours and 32 Minutes ago.

Will do it like this for the DateVersionCommand. :+1: 
Where I would do it more generic in case you want to change this at some point.

My suggestion:
>**Release Time of Version <\<Version\>>:
><\<here some formatted time\>>**
>That was approximately <\<here some formatted duration\>> ago.

Is that okay for you? I just want it easy adjustable but if you say that this won't ever be changed, I will do it like you suggested it.

Other question: Here a function from the JenkinsChangelogProvider class:
```JAVA
    private static int extractBuild(String version)
    {
        int i = version.lastIndexOf('_');
        try
        {
            return Math.max(0, Integer.parseInt(version.substring(i + 1)));
        }
        catch(NumberFormatException ex)
        {
            return 0;
        }
    }
```
Would I be allowed to make it public? I could need that function for the getting the buildtime in the ChangelogCommand since the Changelog just provides the full version and sometimes it could be impossible to fetch a Build Number from that (higher failure percentage). Else I would make a copy of that function in the ChangelogCommand class.

### Current State
DateVersionCommand:
![DateVersionCommand](http://puu.sh/DNV23/b71740a008.png)

ChangelogCommand:
![ChangelogCommand](http://puu.sh/DNUVM/d3b4223bc9.png)

Reply:
Sure, mine was only an idea. You don't have to do it exactly like that.

Reply:
> Sure, mine was only an idea. You don't have to do it exactly like that.

Well normally I expect that the devs have the last word on how they want the code to be and not the people requesting their code to be merged. :D

>Would I be allowed to make it public?

Any info on that? Then I can commit it real quick before going to bed and you can do further code checks. :D

Reply:
I have never touched that piece of code but I don't see any reason why you could make it public

Reply:
Alright, then I will make a copy and don't make it public.

Reply:
*couldn't, I meant I don't see any reason why you couldn't make it public, sry

Reply:
No worries, I already thought that, that's why I actually posted that comment. :D
And even if I would have used a copy, another commit and it's fixed so no worries. :)

Will be committed in the next 5 minutes.